File: parent335/pinyougou-interface/src/main/java/cn/itcast/core/service/order/OrderService.java
package cn.itcast.core.service.order;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.order.Order;

import java.util.List;

public interface OrderService {

    /**
     * 提交订单
     * @param order
     */
    public void add(String username, Order order);

    public List<Order> findAll();

    public PageResult findPage(Order order, Integer page, Integer rows);

    public Order findById(Long orderId);

    /**
     * 分页显示
     * @param page
     * @param rows
     * @return
     */
    PageResult findByPage(Integer page, Integer rows);
}

File: parent335/pinyougou-web-user/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../order/findAll.do');
    }
    this.findByPage = function(page,rows){
        return $http.get('../order/findByPage.do?page='+page+'&rows='+rows);
    }
    this.findStatus = function(){
        return $http.get('../order/findStatus.do?page='+page+'&rows='+rows);
    }
    //增加
    this.add=function(entity){
        return  $http.post('../user/add.do',entity );
    }
    //修改
    this.update=function(entity){
        return  $http.post('../user/update.do',entity );
    }

});